### DISCUSSION TITLE
Any access to h5 and h6 in editor ?

### DISCUSSION THREAD

**Discussion by Limezy** [outline/outline#3514] [General]
Hi !
I have noticed that Outline is capable of manipulating h4, h5 and h6 headers when importing them form a markdown file.
I have also noticed that there is a hidden shortcut ⌃⇧4 that activates a h4 header in the editor.
Is there any way to create a h5 or h6 header in the editor ? If not, even if hidden behind a specific shortcut, that would be useful !

**Comment by tommoor** [comment]
⌃⇧1-6 should all work, you can also use markdown and add ##### at the beginning of a line.

**Reply by Limezy** [reply]
Let me double check because I have tried both options without success
It's maybe because of my unusual browser (Orion)

**Reply by Limezy** [reply]
I confirm that it doesn't work with other browsers. Looks like a lost feature ? 
Shall I create a bug report from the discussion ?

**Reply by tommoor** [reply]
Hmmm - I think we removed on purpose at some point because anything past 4 looked just like bold text anyway. 

It's also too many levels to keep in your head, there is such a thing as too much hierarchy 😂

**Reply by almereyda** [reply]
Yes (: In wiki-pattern-land we call this : Triple Meta Antipattern : http://wiki.c2.com/?TripleMeta

**Reply by Limezy** [reply]
I fully agree. Actually my intention was not to use it as a hirerachy but more as a hack for my notes. The idea was to have a back and forth header link so that when you click on the note number, you are moved to the note, and then you have a backlink to come back. In this regard I was looking for something which would have an anchor (= a header) but that would look as close as possible from normal text

**Reply by Limezy** [reply]
https://user-images.githubusercontent.com/24638389/167841921-57c41a87-8abf-4436-92f3-f9a64d41eca8.mov

Demo of how I use these anchors for my document small notes

**Reply by tommoor** [reply]
well, it looks great!

**Reply by Limezy** [reply]
Yes, very close to what I need ! But... 
1. It's quite long to create manually all the header links
2. It's fragile because if I add a note between note 1 and note 2, I need to recreate all the links (the link will not update itself when the header is changed)
3. It's broken in share mode because the url is not exactly the same. There is something like /doc/ vs /share/

**Reply by almereyda** [reply]
This seems like a very advanced design sketch for implementing footnotes https://github.com/outline/outline/discussions/2582.

@Limezy Does the workflow described above continue and result from the experiments/-periences in #2582?

**Reply by Limezy** [reply]
> This seems like a very advanced design sketch for implementing footnotes #2582.
> 
> @Limezy Does the workflow described above continue and result from the experiments/-periences in #2582?

Thanks for your help ! 

Yes, these are also my former experiments. The technique I describe in #2582 was slightly more robust, as it was designed around the document nesting and automatic linking. As a result, when I changed the title of my note I could be sure that the footnotes links wouldn't be broken. However I was ending up with a lot of nested documents which looked quite bad and were confusing for some of my readers which are not very digital natives... 

This is why I ended up having my footnotes as H4 headers at the bottom of each document, add a ⤴︎ unicode at the end of each, and then make both links manually using the `mydomain.tld/doc/title-of-my-doc-XD3WoBsWom#h-note-¹` format
